app/utils/estimate_prioritization/prioritization_criteria.py

```diff
@@ -53,8 +52,11 @@
     ],
     'test_type': [
         lambda estimate: estimate['test_type'] == 'Neutralization',
+        lambda estimate: estimate['test_type'] == 'Multiple Types',
+        lambda estimate: estimate['test_type'] == 'ELISA',
         lambda estimate: estimate['test_type'] == 'CLIA',
-        lambda estimate: estimate['test_type'] == 'ELISA'
+        lambda estimate: estimate['test_type'] == 'LFIA',
+        lambda estimate: estimate['test_type'] == 'Other',
```

Comment:
These 'catch-call' cases are already implied to be the last thing in the hierarchy, if they're not listed among the options  listed- but it's good to be explicit about them!